cursorless-talon-dev/src/cursorless_dev_vscode.py

```diff
@@ -0,0 +1,28 @@
+from talon import Context, Module
+
+mod = Module()
+ctx = Context()
+
+ctx.matches = r"""
+app: vscode
+win.title: /cursorless/
+"""
+
+
+mod.list(
+    "cursorless_launch_configuration", "A launch configuration for Cursorless vscode"
+)
+
+ctx.lists["user.cursorless_launch_configuration"] = {
+    "stench": "Run extension",
+    "test": "Extension tests",
+    "test subset": "Extension tests subset",
+    "test unit": "Unit tests only",
+    "test talon": "Talon tests",
+    "test talon subset": "Talon tests subset",
+    # "Update fixtures": "Update fixtures",
+    # "Update fixtures subset": "Update fixtures subset",
+    # "Docusaurus start": "Docusaurus start",
+    # "Docusaurus build": "Docusaurus build",
+    # "cursorless.org client-side": "cursorless.org client-side",
```

Comment:
Some of these seemed to dangerous / niche to map

Reply:
Not sure if I'm a fan of these tors terse launch commands

Reply:
Most of them mirror the launch name pretty closely. Are you referring to "stench"?

Reply:
To be honest the stench one is a bit unintuitive

Reply:
would be nice if it were configurable, but I'm not sure how to do that easily. I guess we could just have the slightly more verbose version and contributor can map separate short command if they want it?

Reply:
That sounds reasonable.